plexpy/webserve.py

```diff
@@ -24,6 +24,7 @@
 from io import open, BytesIO
 import base64
 import json
+import ssl
```

Comment:
I had another thought. This can be `import ssl as _ssl` then `_ssl.CERT_NONE` below so that no API changes need to be made.

I will push these changes myself.